bigtable/client/rpc_backoff_policy_test.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/rpc_backoff_policy.h"
+#include "bigtable/client/chrono_literals.h"
+
+#include <gtest/gtest.h>
+#include <chrono>
+
+namespace {
+/// Create a grpc::Status with a status code for transient errors.
+grpc::Status transient() {
```

Comment:
`TransientError()` ?

Reply:
CreateTransientError in view of your latter comment.